app/views/users/index.html.erb

```diff
@@ -7,8 +7,19 @@
 		<li>
 			<%= gravatar_for user %> 
 			<%= link_to user.name, user %> 
+<<<<<<< bf72d7571f74e7046221dd1d124ef59974607202
+      <% if current_user.admin? && current_user !== user) %>
+=======
+      <% if current_user.admin? && current_user != user %>
+>>>>>>> Password resets are currently functional and classes have been refactored.
```

Comment:
And I'll look into the integration test stuff to catch these kinds of abominations...